src/stripe.core.ts

```diff
@@ -142,7 +156,7 @@ export function createStripe(
     this._setApiKey(key);
 
     this.errors = _Error;
-    this.webhooks = createWebhooks(platformFunctions);
+    this.webhooks = Stripe.webhooks;
```

Comment:
Any reason why we're changing `webhooks` on the `Stripe` instance as well? Feels a little wrong to let this one start also act as a factory function when we're removing that behavior later - we'd prefer to keep the breaking change surface minimal when the time comes.

Reply:
Ah yes you're right I was getting ahead of myself 😉